assets/css/admin/upgrades/_v3.scss

```diff
@@ -0,0 +1,15 @@
+#edd-migration-progress {
+	.dashicons-yes {
```

Comment:
Could we tweak the color on the ellipses dashicon to be more of an "inactive" color? Just a lighter grey?